dashboard/config/scripts/levels/2-3 Bee Debugging 3.level

```diff
@@ -31,7 +31,6 @@
     "definition_highlight": "false",
     "definition_collapse": "false",
     "disable_examples": "false",
-    "maze_data": null,
```

Comment:
Looks like this extra diff got added in these four commits: https://github.com/code-dot-org/code-dot-org/commit/ec21fd866568219e0736964cae456df9d65093de, https://github.com/code-dot-org/code-dot-org/commit/07ef62e57870f85fd50af9547801e3c08e46b2d1, https://github.com/code-dot-org/code-dot-org/commit/3d747fda520af66b863acedef636014a69bee476, https://github.com/code-dot-org/code-dot-org/commit/aa1f08fb143712bb2c485c88313d772c1a1bb176.